utoipa-gen/Cargo.toml

```diff
@@ -43,6 +43,7 @@ smallvec = { version = "1.10", features = ["serde"] }
 rust_decimal = { version = "1", default-features = false }
 chrono = { version = "0.4", features = ["serde"] }
 assert-json-diff = "2"
+assert_matches = "1.5.0"
```

Comment:
Perhaps there is no need for additional dependency? :thinking: Even making this kind of macro by hand should be quite trivial. Or perhaps there is no need to even a macro. Just typing `match {}` with `panic` on any branch.

Reply:
You're right. I'll modify the test I added.

[`assert_matches!()`](https://doc.rust-lang.org/std/?search=assert_matches) is one of standard macros, but available only in nightly at this point (will be stabilized in the future). So, I usually add the [`assert_matches`](https://crates.io/crates/assert_matches) crate and use the same name macro in my tests by habit.

Reply:
modified in 4c501ce97105e23a99bc43168e8845fa16942961

Reply:
Yup, seems a good crate though, I just try to keep the dependencies as minimal as possible for what comes to the library. Though eventually when utoipa will use the minimum rust version containing this macro it can be naturally used. Now though for users we need to support rust some generations backwards.